recipes/singularityce/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -xeuf
+
+# Workaround for travis permissions errors which end up creating very verbose log output
+if [[ $(uname -m) == "ppc64le" ]]; then
+    export GOPATH=${TMPDIR:-/tmp}/gopath
+    mkdir -p "${GOPATH}"
+fi
+
+pushd src/github.com/sylabs/${PKG_NAME}
+
+# The "starter" binary inherits the stack from "singularity" meaning FORTIFY_SOURCE cannot be used
+CGO_CPPFLAGS=$(echo "${CGO_CPPFLAGS}" | sed -E 's@FORTIFY_SOURCE=[0-9]@FORTIFY_SOURCE=0@g')
+export CGO_CPPFLAGS
+
+# configure
+./mconfig \
+  -P release-stripped \
+  --without-suid \
+  -p "${PREFIX}" \
+  -c "${CC}" \
+  -x "${CXX}"
+
+# build
+pushd builddir
+export LD_LIBRARY_PATH=${PREFIX}/lib
+make
+
+# install
+make install
+
```

Comment:
Is there a `make check` we can run here?

Reply:
Hi @ocefpaf! I don't think so... or at least I haven't seen it. 

For reference, 

1. this is our original singularity here at conda-forge: [link](https://github.com/conda-forge/singularity-feedstock/blob/7ec89a8b71fa136d66f46367101890fcd2c1990f/recipe/build.sh#L24-L30)
2. and this is the dev install docs: [link](https://sylabs.io/guides/latest/admin-guide/installation.html#:~:text=%24%20./mconfig%20%2D%2Dwithout%2Dsuid%20%2D%2Dprefix%3D/home/dave/singularity%2Dce%20%26%26%20%5C%0A%20%20%20%20make%20%2DC%20./builddir%20%26%26%20%5C%0A%20%20%20%20make%20%2DC%20./builddir%20install)

Do you think we should be adding a `make check` here as well?

Reply:
If it is available, yes. I'll leave that to you in the feedstock.